openedx/core/djangolib/tests/test_js_utils.py

```diff
@@ -10,6 +10,9 @@
 
 from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
 
+from django.utils.translation import ugettext_lazy
+from xmodule.modulestore import EdxJSONEncoder
```

Comment:
I don't like that we're adding tests to "test_js_utils.py" that are testing code from elsewhere, especially when elsewhere is modulestore.  I know it's a hassle to refactor things, but we need to try to keep the tree making sense.

Reply:
Re-factored and moved to **`common/lib/xmodule/xmodule/tests/test_export.py`** because this file contains the tests for the **`EdxJSONEncoder`**